.github/scripts/releasecli.sh

```diff
@@ -1,9 +1,10 @@
+GITHUB_SHA=$1
```

Comment:
Just realised that the `releasecli.sh` is the only shellscript that doesn't specify that is a bash script.

That is probably a mistake done by my part in a previous story but is interesting to fix this.